src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/helpers/deriveAutoGenerateEvents.js

```diff
@@ -40,10 +40,12 @@ export const deriveAutoGenerateEvents = ({
         .map(
             ({
                 id: programStage,
+                reportDateToUse,
                 generatedByEnrollmentDate: generateScheduleDateByEnrollmentDate,
                 openAfterEnrollment,
                 minDaysFromStart,
             }) => {
+                const reportDate = reportDateToUse === 'enrollmentDate' ? enrolledAt : sanitizedOccurredAt;
```

Comment:
There are cases where reportDateToUse is set to either `'enrollmentDate'` or `'enrolledAt'`. I believe this inconsistency arises because it's stored as `'enrolledAt'` in the [ProgramStage](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/metaData/Program/ProgramStage.js#L168). Would it be possible to standardize the reportDateToUse value on the client side? Thanks!